js/apps/system/_admin/aardvark/APP/frontend/js/views/viewsView.js

```diff
@@ -312,21 +316,24 @@
         )
       );
 
-      tableContent.push(
+      storedValuesTableContent.push(
         window.modalView.createTableEntry(
           'newStoredValues',
-          'Stored Values',
-          ['Fields (one per line)', 'Compression'],
+          {
+            noLabel: true // Dummy object. Entry will only be created if this is set as a String.
+          },
+          ['Fields', 'Compression'],
           [[
-            window.modalView.createBlobEntry(
+            window.modalView.createSelect2Entry(
```

Comment:
No option to reorder contents except delete/re-insert?

Reply:
Due to technical issues with the old framework, this feature should be added in our upcoming refactoring using the react js framework. This feature is planned for 3.10. The required time for implementing this feature in 3.9 (using backbone) only, and then drop it in 3.10 anyway, is not worth it. Additionally, a 3.9 backbone based implementation would have been super ugly, from a technical point of view. Also, the required time is out of proportion compared to the benefits it would deliver here.